_includes/post-macros.html

```diff
@@ -291,8 +291,8 @@ <h1 class="summary_heading">{{ post.title|safe }}</h1>
     'active_filters_total': active_filters_total
 } -%}
 
-{%- import 'macros/util/is-mobile.html' as is_mobile -%}
```

Comment:
I apologize for being late to the party here-- but this stuff makes me a little nervous: template code fires on the server, which means the output will be cached by Akamai and served to potentially many devices.

There might be solutions here ([perhaps, instructing Akamai to honor Vary: headers, and sending Vary:user-agent](https://www.fastly.com/blog/best-practices-for-using-the-vary-header/)), so don't panic yet.